TESTING.md

```diff
@@ -119,13 +119,25 @@ Prerequisite: You have to be **signed in**. This is important because we expect
 
 ## Commands
 
-### General commands availability in Cody toolbar
+### General commands availability in Cody tool window
 
-1. Navigate to `Cody` toolbar and open `Commands`.
+1. Navigate to `Cody` tool window and open `Commands`.
 
 #### Expected behaviour
 
-* List of commands is immediately available after the toolbar is displayed. **No refresh is required.**
+* List of commands is immediately available after the tool window is displayed. **No refresh is required.**
+
+### General commands availability from keyboard shortcuts
+
+| Command       | Windows / Linux                                      | MacOs                                                |
+|---------------|------------------------------------------------------|------------------------------------------------------|
+| Explain Code  | <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>1</kbd>     | <kbd>control</kbd> + <kbd>Shift</kbd> + <kbd>E</kbd> |
+| Smell Code    | <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>2</kbd> | <kbd>control</kbd> + <kbd>Shift</kbd> + <kbd>S</kbd> |
+| Generate Test | <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>3</kbd> | <kbd>control</kbd> + <kbd>Shift</kbd> + <kbd>T</kbd> |
+
+#### Expected behaviour
+
+* Commands work as executed from any other place (the tool window opens if needed).
 
```

Comment:
I think we should add some info mnemonics and how this works/how to test (it was not obvious for me).

Reply:
Let me explain. 

Mnemonics are a common solution applied in IntelliJ and its plugins (and probably many other tools. They help to "click" buttons and choose options without the mouse (or clicking TAB key). Having the focus on some window you click `option` on macOS (idk what is the key on Linux) - having the key pressed you see the buttons have underlined characters - ideally each button has a different character. Then you can click the specific (underlined) key to "click" the given button. 

Try it in IntelliJ's commit panel. Pressing `option` there underlines `m` in `Amend`, `i` in `Commit` and `P` in `Commit and Push...`. 

---

I do not think we need to describe it here. It is very simple. Only nerds use that 🤓 It was not requested officially. It's just a nice trick useful for ones that like to use it.